foundry.toml

```diff
@@ -4,7 +4,10 @@ out = 'out'
 build_info = true
 build_info_path = 'out/build-info'
 solc_version = "0.8.19"
+evm_version = 'cancun'
```

Comment:
@arcantheon 
please add a comment on why we had to add this config

Reply:
Added be8da3709f7ae67582554137f55d693800b8641d